src/confetti.js

```diff
@@ -190,7 +190,9 @@
     ],
     // probably should be true, but back-compat
     disableForReducedMotion: false,
-    scalar: 1
+    scalar: 1,
+    disableWobble: false,
+    wobbleSpeed: Math.random() * 0.1 + 0.05
```

Comment:
This is not equivalent to the previous value of `Math.min(0.11, Math.random() * 0.1 + 0.05)`. When this value is too small, the animation does not look right, and I am not sure why it was changed.